scripts/package/package_win32_sdk.sh

```diff
@@ -60,12 +60,14 @@ if [ ! -e "${TARGET_PATH}/${PACKAGES_WIN32_TOOLCHAIN}" ]; then
 	TMP=MicrosoftVisualStudio2019
 
```

Comment:
Not related to this PR, but VC_PATH above assumes Community edition, we should mention in a readme you need to change this if you have Professional. Or even better detect it :)